<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Simulator</class>
 <widget class="QMainWindow" name="Simulator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionClose"/>
   <addaction name="actionLoad"/>
   <addaction name="actionCameraImage"/>
   <addaction name="separator"/>
   <addaction name="actionRun"/>
   <addaction name="actionStop"/>
  </widget>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../assets/icons.qrc">
     <normaloff>:/prefix/cross.png</normaloff>:/prefix/cross.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close this window</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="../assets/icons.qrc">
     <normaloff>:/prefix/load.png</normaloff>:/prefix/load.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>Load a config</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="../assets/icons.qrc">
     <normaloff>:/prefix/arrow.png</normaloff>:/prefix/arrow.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="toolTip">
    <string>Run the simulator</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../assets/icons.qrc">
     <normaloff>:/prefix/stop.png</normaloff>:/prefix/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop the simulator</string>
   </property>
  </action>
  <action name="actionCameraImage">
   <property name="icon">
    <iconset resource="../assets/icons.qrc">
     <normaloff>:/prefix/camera.png</normaloff>:/prefix/camera.png</iconset>
   </property>
   <property name="text">
    <string>CameraImage</string>
   </property>
   <property name="toolTip">
    <string>Select an image for camera</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../assets/icons.qrc"/>
 </resources>
 <connections/>
</ui>
